client/blocks/inline-help/inline-help-center-content.scss

```diff
@@ -113,51 +102,78 @@
 	}
 }
 
-.inline-help__footer {
-	background: var( --color-neutral-0 );
-	border-top: 1px solid var( --color-neutral-10 );
-	border-radius: 0 0 2px 2px;
-	display: flex;
-	flex-direction: row;
-	justify-content: space-between;
-	flex-wrap: wrap;
-
-	.button.is-borderless.inline-help__more-button,
-	.button.is-borderless.inline-help__contact-button,
-	.button.is-borderless.inline-help__cancel-button {
-		text-align: left;
-		position: relative;
-		padding: 12px 8px 12px 46px;
-
-		.gridicon.inline-help__gridicon-left {
-			margin: 0;
-			position: absolute;
-			top: 11px;
-			left: 13px;
-		}
```

Comment:
Removed the footer CSS, as we don't have a footer here, but we use the help-center component one.